test/features/parsoid/transform.js

```diff
@@ -1,8 +1,9 @@
 'use strict';
 
-const assert = require('../../utils/assert.js');
 const Server = require('../../utils/server.js');
 const preq   = require('preq');
```

Comment:
I don't think you are using this anymore, right?

Reply:
Nope, `preq` has now been totally removed from `transform.js`.